test/file1-1.tsx

```diff
@@ -0,0 +1,129 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { Collapse } from '@xsite-ui/common.collapse';
```

Comment:
Importing 'Collapse' from '@xsite-ui/common.collapse' but it is not used in the code.